packages/frontend/src/view/components/protocol/LayerZeroMultisig.tsx

```diff
@@ -1,39 +1,177 @@
+    return (
+      <ProtocolComponentCard
+        title="LayerZero Multisig"
+        subtitle="Data could not be loaded ⚠️"
+      />
+    )
+  }
+
+  if (!transactions || transactions.length === 0) {
+    return (
+      <ProtocolComponentCard
+        title="LayerZero Multisig"
+        subtitle="No transactions executed"
+      />
+    )
+  }
+
   return (
-    <Card subtitle={subtitle}>
+    <ProtocolComponentCard
+      title="LayerZero Multisig"
+      subtitle={subtitle}
+      description="Safe multi-signature contract managed by LayerZero. Owner of Endpoint and UltraLightNodeV2 contracts. Any configuration change must be made through the LayerZero multisig wallet. Transaction information is being fetched from safe transaction service."
+    >
       {hasData && (
         <>
-          <Row label="Threshold" value={`${threshold}/${owners.length}`} />
-          <Row
-            label="Owners"
-            value={<Code>{JSON.stringify(owners, null, 2)}</Code>}
-          />
+          <Subsection>
+            <Row label="Threshold" value={`${threshold}/${owners.length}`} />
+            <RowSeparator />
+            <Row
+              label="Owners"
+              value={
+                <div className="flex flex-col gap-1 text-center text-3xs md:gap-5 md:text-left md:text-xs">
+                  {owners.map((owner, i) => (
+                    <span key={i}>{owner}</span>
+                  ))}
+                </div>
+              }
+            />
+          </Subsection>
+          <Subsection>
+            <div className="flex flex-col items-center justify-between gap-3 py-3 text-md font-medium md:flex-row md:gap-0">
+              <h1>Multisig transactions</h1>
+              <PaginationControls
+                amountOfPages={TOTAL_PAGES_AMOUNT}
+                currentPage={page}
+                setPage={setNormalizedPage}
+              />
+            </div>
+
+            <div className="overflow-x-auto">
+              <div className="col-span-5 grid min-w-[800px] grid-cols-multisig rounded bg-gray-300 py-3 text-left text-[13px] font-semibold text-[#AEAEAE]">
+                <div className="px-6">SUBMITTED</div>
+                <div>METHOD</div>
+                <div>CONFIRMATIONS</div>
+                <div>STATUS</div>
+                <div />
+              </div>
+              <PaginatedContainer itemsPerPage={TXS_PER_PAGE} page={page}>
+                {transactions.map((tx, i) => (
+                  <SafeMultisigTransactionComponent
+                    amountOfOwners={owners.length}
+                    tx={tx}
+                    allTxs={transactions}
+                    key={i}
+                  />
+                ))}
+              </PaginatedContainer>
+            </div>
+
+            <span className="pb-3 pr-8 pt-5 text-right text-xs text-gray-15">
+              {LOWER_PAGE_BOUND} - {UPPER_PAGE_BOUND} out of{' '}
+              {transactions.length} transactions
+            </span>
```

Comment:
Let's move it below (or above?) the pagination buttons

Reply:
[feat: move page summary below controls](https://github.com/l2beat/lz-monitoring/pull/65/commits/8767a07554c0eddd45a61c1c27db22e054926407)